core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -183,7 +183,7 @@
       logger.debug("Exit (OK): {}", callable);
       return result;
     } catch (Exception e) {
-      logger.debug("Exit (Error): {}, Error={}", callable, e.getMessage());
+      logger.warn("Exit (Error): {}, Error={}", callable, e.getMessage());
```

Comment:
Could you change this to `debug`, we try to keep symmetry for the enter/exit logging.